final_task/rss_reader/cmd_line.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import argparse
+import logging
+from final_task.rss_reader import rss_reader
```

Comment:
This package import is redundant here. Think about where exactly `import` statements starts to search for specified module.